database_init_SQL.txt

```diff
@@ -95,7 +98,7 @@ Important Notes:
         FOR EACH ROW EXECUTE PROCEDURE set_qty_remaining();
 
     -------------------------------------------------------------------
-    CREATE OR REPLACE FUNCTION max_item_id() 
+    CREATE OR REPLACE FUNCTION max_item_id()
```

Comment:
Is this a reliable way of getting the item id?  I think it would work as long as your item was the last item inserted into the table.  be careful here as you could have a sequence of insert1, insert2, add_tag1, add_tag2 and both tags would be added to item2.